larray/core/array.py

```diff
@@ -3145,6 +3145,282 @@ def indicesofsorted(self, axis=None, ascending=True, kind='quicksort'):
 
     posargsort = renamed_to(indicesofsorted, 'posargsort')
 
+    # TODO: move some doctests to unit tests
+    # TODO: implement keys_by
+    # TODO: implement expand=True
+    def keys(self, axes=None, ascending=True):
+        r"""Returns a view on the array labels along axes.
+
+        This is an object you can iterate (loop) on or index to get to the Nth label along axes.
```

Comment:
duplicate of section Returns.
You didn't repeated the sentence in the documentation of values() below.

Reply:
fixed